etna/transforms/math/sklearn.py

```diff
@@ -59,15 +59,14 @@ def __init__(
         ValueError:
             if incorrect mode given
         """
-        super().__init__(required_features="all")
-        if inplace and (out_column is not None):
-            warnings.warn("Transformation will be applied inplace, out_column param will be ignored")
-
-        self.transformer = transformer
-
         if isinstance(in_column, str):
             in_column = [in_column]
+        super().__init__(required_features=sorted(in_column) if in_column is not None else "all")
```

Comment:
Looks difficult for one line. Move expression in `__init__` on the line above.

Reply:
Fixed